apps/mull-api/src/app/event/location-autocomplete/location-autocomplete.resolver.ts

```diff
@@ -0,0 +1,18 @@
+import { Query, Args } from '@nestjs/graphql';
+import axiosInstance from '../../../../axiosConfig';
+
+export class LocationAutocompleteResolver {
+  @Query(/* istanbul ignore next */ () => [String])
+  async getAutocompletedLocations(@Args('userInput') userInput: String) {
+    const response = await axiosInstance.request({
+      method: 'get',
+      url: `https://maps.googleapis.com/maps/api/place/autocomplete/json?input=${userInput}&key=sadas`,
+    });
+    const autocompleteLocations = response.data.predictions.map((field) => field.description);
+
+    if (response.data.error_message) {
+      throw new Error(response.data.error_message);
+    }
```

Comment:
Really minor, but it doesn't make sense to process the response data before checking for error, the order of these two statements should be switched
```suggestion
    if (response.data.error_message) {
      throw new Error(response.data.error_message);
    }
    
    const autocompleteLocations = response.data.predictions.map((field) => field.description);
```